AndroidManifest.xml

```diff
@@ -267,11 +263,13 @@
             android:name=".ui.activity.ManageSpaceActivity"
             android:label="@string/manage_space_title"
             android:theme="@style/Theme.ownCloud" />
-        <activity android:name=".ui.activity.PrivacyPolicyActivity"></activity>
-        <activity android:name=".ui.preview.PreviewVideoActivity"
+        <activity android:name=".ui.activity.PrivacyPolicyActivity" />
+        <activity
+            android:name=".ui.preview.PreviewVideoActivity"
             android:configChanges="keyboard|keyboardHidden|orientation|screenSize|screenLayout|smallestScreenSize|uiMode"
-            android:theme="@style/Theme.ownCloud.Fullscreen">
-        </activity>
+            android:theme="@style/Theme.ownCloud.Fullscreen" />
+        <activity android:name=".ui.activity.CapturedFileChecker" />
+        <activity android:name=".ui.activity.Camera" />
```

Comment:
Same here, **CameraActivity**

Reply:
[comment](https://github.com/owncloud/android/pull/2075#discussion_r159117713) the same applies for the Camera Activity.